services/web/server/src/simcore_service_webserver/catalog/_handlers.py

```diff
@@ -318,3 +320,28 @@ async def get_service_resources(request: Request):
     return await asyncio.get_event_loop().run_in_executor(
         None, envelope_json_response, data
     )
+
+
+@routes.get(
+    f"{VTAG}/catalog/services/{{service_key}}/{{service_version}}/pricing-plan",
+    name="get_service_pricing_plan",
+)
+@login_required
+@permission_required("services.catalog.*")
+async def get_service_pricing_plan(request: Request):
+    ctx = CatalogRequestContext.create(request)
+    path_params = parse_request_path_parameters_as(ServicePathParams, request)
+
+    service_pricing_plan: ServicePricingPlanGet = (
+        await get_default_service_pricing_plan(
+            app=request.app,
+            product_name=ctx.product_name,
+            service_key=path_params.service_key,
+            service_version=path_params.service_version,
+        )
+    )
+
+    data = service_pricing_plan.dict(**RESPONSE_MODEL_POLICY)
```

Comment:
You do not need:
- convert to dict here  (this is already done inside `envelope_json_response` )
- call `asyncio.get_event_loop` ...


The trick of the `event_loop` was introduced by @sanderegg to avoid blocking while pydantic is validating. Here it has no effect since value passed is already a model and therefore it will not be re-validated!

Reply:
I did it to be in consistency all the other in this file, but i can change it